shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Genesis.hs

```diff
@@ -20,12 +20,14 @@ module Shelley.Spec.Ledger.Genesis
     initialFundsPseudoTxIn,
     validateGenesis,
     describeValidationErr,
+    mkShelleyGlobals,
   )
 where
 
 import qualified Cardano.Crypto.Hash.Class as Crypto
 import Cardano.Crypto.KES.Class (totalPeriodsKES)
-import Cardano.Prelude (NoUnexpectedThunks, forceElemsToWHNF)
+import Cardano.Prelude (Identity, Natural, NoUnexpectedThunks, forceElemsToWHNF)
```

Comment:
Please don't import things from Cardano.Prelude other than the `NoUnexpectedThunks` stuff. As soon as that gets moved out (https://github.com/input-output-hk/cardano-prelude/issues/115) we will drop use of `cardano-prelude`